src/html/partials/story/intro.hbs

```diff
@@ -216,47 +225,56 @@ TODO: more examples, quick slideshow. Is this song actually atypical? Why? If no
 
 <div class="step" data-song="Royals">
 <p>
-Lorde burst on the scene in 2013 with her singular debut single, <i>Royals</i>.
+	Lorde burst on the scene in 2013 with her debut single, “Royals.”
 </p>
 
 <p>
-<i>Royals</i> has been described as "minimalist pop", and its sonic attributes match that description well.
+	“Royals” has been described as “minimalist pop,” and its sonic attributes, all with the exception of dancability dramatically pared down compared to the previous year’s average, match that description well.
```

Comment:
I like this extra explanation, but I think it makes the sentence a little unwieldy/hard to parse. Maybe the "pared down" part could be a separate sentence?

Reply:
Works for me